arabic/salamoAlikom.md

```diff
@@ -9,3 +9,5 @@
 [اريد ان اتعلم المزيد](learn/learn.md)
 
 [اريد ان احلم](dream/dream.md)
+
+(passion/passion.md) [اريد ان اجد شغفي في الحياه]
```

Comment:
It looks like you intended this to be a link, but it needs to go in the other order (like the links above). So the text in square brackets should come first, and the link in parentheses should follow.